pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -86,6 +87,7 @@ func Run() {
 	debugPtr := flag.Bool("d", false, "Debug flag")
 	curpartPtr := flag.String("c", "", "Current partition")
 	flag.Parse()
+	dnsServers = make(map[string][]net.IP)
```

Comment:
I don't understand why we need a global here; I think there can be a different set on eth0 and eth1 when eth1 is a shopfloor network and eth0 is the Internet.

Shouldn't we track this in DeviceNetworkStatus?

Reply:
Yes, ctx status has this 'old' status, but this is keep changing. every time the network condition changes, it can be reset to zero, and after regain the connection, have a new one which normally is the same as the previous valid old one. So, this 'global' servers, only track if there is a real DNS server changes. Will put comment in.

Reply:
the map[] has the port-name as the index, so it is tracked per interface here. And the only guy who is interested in this structure is this DnsModify code locally. Thus I would think not not add this in general to the DeviceNetworkStatus part.